### ISSUE TITLE
Enhancement: Option to anchor chat editbox to top/bottom/free drag

### ISSUE THREAD

**Issue by Cruzix** [shagu/pfUI#207]
Make an option to anchor chat editbox to top/bottom of the chat (on top of chat taps) and a 3rd option to enable free dragging.

**Comment by shagu** [comment]
Chat input box is already free movable.

**Comment by Cruzix** [comment]
Yes but it's nearly impossible to have the editbox exactly same height/width as the channel tap. 

Not to mention its very clunky having to move it directly on top of it as you can't choose which window you want to drag so when the editbox is on top of the chat frame and you want to make it look perfect you'll be dragging the chat frame instead and you have to redo the entire thing. Not sure if that makes sense

**Comment by SarekCDL** [comment]
I experienced that problem too.
And it's almost impossible to put the editbox foreground so we can't use them over the chatframe.

An option to stick or integrate the editbox to chatframe with perfect size may be pretty and useful. ;)
